eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/ObjectsIndex.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.objectionary;
+
+import java.net.URL;
+import java.util.Set;
+import org.cactoos.Scalar;
+import org.cactoos.Text;
+import org.cactoos.iterable.Mapped;
+import org.cactoos.scalar.ScalarOf;
+import org.cactoos.scalar.Sticky;
+import org.cactoos.set.SetOf;
+import org.cactoos.text.Split;
+import org.cactoos.text.TextOf;
+
+/**
+ * The objects index that contains all available eo objects.
+ * <a href="https://github.com/objectionary/home/blob/gh-pages/objectionary.lst">Example</a>
+ * @since 0.29
+ */
+final class ObjectsIndex {
```

Comment:
@volodya-lombrozo Do we now need this object since we introduced `OyIndexed`?

Reply:
@mximp I believe, yes. They represent different entities with different behaviour.

Reply:
@volodya-lombrozo In this case it has to implement an interface. So that I can supply for example in-memory impl.